torch/distributed/_tensor/ops/tensor_ops.py

```diff
@@ -359,6 +360,28 @@ def replica_only_strategy(mesh: DeviceMesh, op_schema: OpSchema) -> StrategyType
     return OpStrategy([PlacementStrategy(replicate_spec)])
 
 
+@register_op_strategy(
+    [aten.scatter_.value, aten.scatter.value, aten.scatter_.src, aten.scatter.src],
```

Comment:
This replicate strategy should also work for other reduce variants, e.g. `scatter.reduce`, `scatter_.reduce`, etc.

Reply:
I think these ops are being deprecated, so I'll leave them as separate PRs